prefetch.bs

```diff
@@ -225,19 +228,40 @@ A <dfn export>cross-origin prefetch IP anonymization policy</dfn> has an <dfn ex
   1. Return false.
 </div>
 
+A <dfn export>prefetch controller</dfn> is a [=struct=] which allows managing a prefetch operation. It has the following [=items=]:
+* <dfn for="prefetch controller">state</dfn> (default `"ongoing"`), which is `"ongoing"` or `"aborted"`
+
+  <div class="note">The state is `"ongoing"` even after prefetch is complete, as long as it was not aborted by the author.</div>
+* <dfn for="prefetch controller">fetch controller</dfn> (default null), which is a [=fetch controller=] or null
+* <dfn for="prefetch controller">prefetch record</dfn> (default null), which is a [=prefetch record=] or nulll
+* <dfn export for="prefetch controller">document</dfn>, which is a {{Document}}
+
+  <div class="note">This is the {{Document}} whose [=prefetch buffer=] contains this's [=prefetch controller/prefetch record=], if any.
+
+<div algorithm>
+  To <dfn export for="prefetch controller">abort</dfn> a [=prefetch controller=] |prefetchController|:
```

Comment:
Maybe name it "abort and discard" to make the behavior described in the note clearer.

Reply:
Used "cancel and discard" for consistency with fetch, which uses "abort" to mean not by the user/author, "terminate" to mean by the UA, and "cancel" to mean either.

Reply:
Wow, I never realized Fetch made that distinction. I don't know if I like it... but yeah, consistency is nice.